src/Okta.Sdk/ApplicationsClient.cs

```diff
@@ -57,6 +58,88 @@ public Task<IApplication> CreateApplicationAsync(CreateBasicAuthApplicationOptio
+            => await PostAsync<JsonWebKey>(
+                new HttpRequest
+                {
+                    Uri = "/api/v1/apps/{appId}/credentials/csrs/{csrId}/lifecycle/publish",
+                    Verb = HttpVerb.Post,
+                    Payload = Convert.FromBase64String(base64EncodedCertificateData),
+                    ContentType = "application/pkix-cert",
+                    ContentTransferEncoding = "base64",
+                    PathParameters = new Dictionary<string, object>()
+                    {
+                        ["appId"] = appId,
+                        ["csrId"] = csrId,
+                    },
+                }, cancellationToken).ConfigureAwait(false);
+
+        /// <inheritdoc />
+        public async Task<IJsonWebKey> PublishBinaryCerCertAsync(byte[] certificate, string appId, string csrId, CancellationToken cancellationToken = default(CancellationToken))
+            => await PostAsync<JsonWebKey>(
+                new HttpRequest
+                {
+                    Uri = "/api/v1/apps/{appId}/credentials/csrs/{csrId}/lifecycle/publish",
+                    Verb = HttpVerb.Post,
+                    Payload = certificate,
+                    ContentType = "application/pkix-cert",
+                    PathParameters = new Dictionary<string, object>()
+                    {
+                        ["appId"] = appId,
+                        ["csrId"] = csrId,
+                    },
+                }, cancellationToken).ConfigureAwait(false);
+
+        /// <inheritdoc />
+        public async Task<IJsonWebKey> PublishDerCertAsync(string base64EncodedCertificateData, string appId, string csrId, CancellationToken cancellationToken = default(CancellationToken))
+            => await PostAsync<JsonWebKey>(
+                new HttpRequest
+                {
+                    Uri = "/api/v1/apps/{appId}/credentials/csrs/{csrId}/lifecycle/publish",
+                    Verb = HttpVerb.Post,
+                    Payload = Convert.FromBase64String(base64EncodedCertificateData),
+                    ContentType = "application/pkix-cert",
+                    ContentTransferEncoding = "base64",
+                    PathParameters = new Dictionary<string, object>()
+                    {
+                        ["appId"] = appId,
+                        ["csrId"] = csrId,
+                    },
+                }, cancellationToken).ConfigureAwait(false);
+
+        /// <inheritdoc />
+        public async Task<IJsonWebKey> PublishBinaryDerCertAsync(byte[] certificate, string appId, string csrId, CancellationToken cancellationToken = default(CancellationToken))
+            => await PostAsync<JsonWebKey>(
+                new HttpRequest
+                {
+                    Uri = "/api/v1/apps/{appId}/credentials/csrs/{csrId}/lifecycle/publish",
+                    Verb = HttpVerb.Post,
+                    Payload = certificate,
+                    ContentType = "application/pkix-cert",
```

Comment:
Should we use `application/x-x509-ca-cert` for binary? See request examples here: https://developer.okta.com/docs/reference/api/apps/#request-example-45

Reply:
You're right, yes, we should.  I'll revisit.